src/envoy/utils/filter_state_utils.h

```diff
@@ -22,11 +22,15 @@ namespace envoy {
 namespace utils {
 
 // Data names in `FilterState` set by Path Matcher filter:
-constexpr char kOperation[] =
+constexpr char kFilterStateOperation[] =
     "com.google.espv2.filters.http.path_matcher.operation";
```

Comment:
Just to confirm: The values in these strings are OK? We don't need to specifically include filter_state in the value?

Reply:
The `FilterState` prefix in var names is to make them more specific but as for value, when they are used, it's already known they are filter states.